### PR TITLE
Do not put .git files or ones listed in .gitignore in tarballs

### PR DESCRIPTION
**Pull request by mulkieran**

### CODE REVIEW

**Review by bgurney-rh — APPROVED** [review]

**Review by bgurney-rh — CHANGES_REQUESTED** [review]
One quick change: delete the space between `TAR_CREATE_OPTS` and `=`

(I forget that sometimes; I only caught it because there was a "command not found" error.)

**Comment by bgurney-rh** [comment]
There's more; try this line:
```
TAR_CREATE_OPTS="--exclude-vcs --exclude-vcs-ignores"
```
I'm testing this now, and it seems to have advanced past this line.

File: blackbox/build-and-package.sh [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -e
 
+TAR_CREATE_OPTS="-exclude-vcs --exclude-vcs-ignores"
```
Comment (bgurney-rh) [inline blackbox/build-and-package.sh:4]: The option `--exclude-vcs` requires two hyphens.

(I just checked mine, and it works; when I run "tar tvf" on the stratisd tarball, I no longer see the '.git' subdirectory.)

Comment (mulkieran) [reply blackbox/build-and-package.sh]: grr. force-pushed once more.

**Review by bgurney-rh — APPROVED** [review]

File: blackbox/build-and-package.sh (hunk 1 above)
Comment (bgurney-rh) [reply blackbox/build-and-package.sh]: Ok, now it matches what passes my test.

**Pull request merged**